docs/content/docs/build/containers.md

```diff
@@ -106,12 +115,20 @@ Deleting a container is a service-specific feature, so you should refer to the d
 
 ### attached/detached
 
-Newly created containers begin in a *detached* state. A detached container is not connected to the Fluid service and no
-data is shared with other clients. This is an ideal time to create initial data in your Fluid data model.
+Newly created containers begin in a *detached* state.
+A detached container is not connected to the Fluid service and no data is shared with other clients.
+This is an ideal time to create initial data in your Fluid data model.
+
+An *attached* container, conversely, **is** connected to the Fluid service, and data changes are shared with the other clients.
+An existing container being loaded by a client will always be in an attached state.
```

Comment:
I believe it is possible to detach a container after it is initially loaded, so we might want to specify that here

```suggestion
An existing container being loaded by a client will always initially be in an attached state.
```